src/components/shared/NavCustom.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { Nav } from 'react-bootstrap';
+
+class NavCustom extends Component {
+    handleClick = (e) => {
+        e.target.parentElement.parentElement.childNodes.forEach(item => {
```

Comment:
Interacting with real DOM - out of React philosophy!
You should use other techniques for stuff like that: change class or use styled for example